controllers.py

```diff
@@ -287,23 +281,41 @@ def getFavs():
 def togglePin():
     userID = auth.current_user.get("id")
     favID = request.json.get("favID")
+    imageUrl = request.json.get("imageUrl")
```

Comment:
Should we really need to pass an imageUrl through the url when we call togglePin? Idk if this is the best solution to updating the imageUrl? Where does this url come from? Modifying the recipes image shouldn't be paired with pinning functionality